src/wpc/altsound/altsound_processor.cpp

```diff
@@ -271,6 +271,29 @@ void AltsoundProcessor::init()
 	ALT_DEBUG(0, "END AltsoundProcessor::init()");
 }
 
+// ----------------------------------------------------------------------------
+
+void AltsoundProcessorBase::setGlobalVol(const float vol_in)
```

Comment:
Is there a reason you didn't put this in the base class?  Could be useful for all derived processors

Reply:
See comment above.